test/integration/test_seq_copy_int.py

```diff
@@ -33,6 +33,22 @@
      " --decode-and-evaluate 0",
      "--beam-size 2 --softmax-temperature 0.01",
      True, False, False),
+    # "Vanilla" LSTM encoder-decoder with attention, greedy and skip softmax
+    ("--encoder rnn --decoder rnn --num-layers 1 --rnn-cell-type lstm --rnn-num-hidden 8 --num-embed 4 "
+     " --rnn-attention-type mlp"
+     " --rnn-attention-num-hidden 8 --batch-size 2 --loss cross-entropy --optimized-metric perplexity --max-updates 2"
+     " --checkpoint-frequency 2 --optimizer adam --initial-learning-rate 0.01 --batch-type sentence "
+     " --decode-and-evaluate 0",
+     "--beam-size 2 --softmax-temperature 0.01",
```

Comment:
I don't see the `--skip-softmax` here; also, using --softmax-temperature in this test is quite confusing then :)

Reply:
`--skip-softmax` is not an option anymore, but the default behaviour. Sorry about the confusion; it is something I changed with the third commit.

Reply:
you should update the comment then. Also: It's `--beam-size 2` and therefore not greedy ;)